custom/enikshay/tasks.py

```diff
@@ -67,6 +67,10 @@ def enikshay_task(self):
 
     domains = toggles.UATBC_ADHERENCE_TASK.get_enabled_domains()
     for domain in domains:
+        if toggles.DATA_MIGRATION.enabled(domain):
+            # Don't run this on the india cluster anymore
+            continue
+
```

Comment:
we should do that too but the messages would still queue up in rabbitmq if that's all we did